apps/financial-aid/web-osk/src/routes/application/Info/info.tsx

```diff
@@ -22,12 +22,13 @@ import {
 import { NationalRegistryUserQuery } from '@island.is/financial-aid-web/osk/graphql'
 import { useLogOut } from '@island.is/financial-aid-web/osk/src/utils/hooks/useLogOut'
 import { AppContext } from '@island.is/financial-aid-web/osk/src/components/AppProvider/AppProvider'
+import { useMunicipality } from '@island.is/financial-aid/shared/components'
 
 const ApplicationInfo = () => {
   const router = useRouter()
-  const { user, setMunicipality, setNationalRegistryData } = useContext(
-    AppContext,
-  )
+  const { user, setNationalRegistryData } = useContext(AppContext)
+
+  const { setMunicipality } = useMunicipality()
```

Comment:
setMunicipality can be gotten from appcontext so we don't need to import the useMunicipality hook.